src/explore-education-statistics-admin/src/pages/bau/BauDashboardPage.tsx

```diff
@@ -38,6 +39,18 @@ const BauDashboardPage = () => {
             View a list of all publication contacts.
           </p>
         </div>
+        {canManageImports && (
```

Comment:
This can be inlined more succinctly to:

```javascript
user?.permissions.canAccessAllImports && (...)
````

The same applies to the other permission checks as well.